supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -128,31 +145,28 @@ See xref:container[container] in the _General Conventions_ part.
 *See also*:
 
 [discrete]
-[[deployment-configuration]]
-==== DeploymentConfig (noun)
-*Description*: A DeploymentConfig is an API object that contains the details of a
-particular application deployment: the configuration used in the replication
-controller definition, such as the number of replicas to ensure; triggers for
-automatically performing an updated deployment, such as when an image is tagged
-or the source code in a source-to-image deployment is changed; the strategy for
-transitioning between deployments when upgrading; and lifecycle hooks.
+[[custom-resource-definition]]
+==== custom resource definition (noun)
+*Description*: A custom resource definition (CRD) defines a new, unique object `Kind` in the cluster and lets the Kubernetes API server handle its entire lifecycle.
 
 *Use it*: yes
 
 *Incorrect forms*:
 
-*See also*: xref:api-objects[API objects]
+*See also*:
 
 [discrete]
-[[deployment-controller]]
-==== deployment controller (noun)
-*Description*: A Kubernetes object that creates a replication controller from a given Pod template. If that Pod template is modified, the deployment controller creates a new replication controller based on the modified Pod template and replaces the old replication controller with this new one.
+[[deployment]]
+==== deployment (noun)
+*Description*: A statement of intent by a user to roll out a new version of a configuration. A replication controller can be used for that and other purposes. All deployments are replication controllers, but not all replication controllers are deployments. To avoid confusion, do not refer to an overall OpenShift installation, instance, or cluster as an "OpenShift deployment".
```

Comment:
"A replication controller can be used for that and other purposes. All deployments are replication controllers, but not all replication controllers are deployments."

These (old) sentences are strange/vague (first sentence) or not worth mentioning (second sentence) to me, but also leaves out replica sets, which at this point are more common than replication controllers AFAIK (since Deployments are preferred to DeploymentConfigs). I might just remove them, and possibly also add a sentence that asscoiates Deployments with ReplicaSets and DeploymentConfigs with RelicationControllers (I notice now that "replication controller" has an entry but "replica set" does not).

Reply:
Thanks. Agree - I've removed those two sentences. And below where I mention the two deployment objects, I added whether they use replication controllers or replica sets.